modules/vm/templates/entrypoint.sh.tpl

```diff
@@ -14,9 +14,102 @@ done
 # Login in Azure CLI with managed identity (user or system assigned)
 az login --identity
 
+# Get the license
+az keyvault secret download --vault-name ${key_vault_name} --name graphdb-license --file /etc/graphdb/graphdb.license --encoding base64
+
 # TODO: Find/create/mount volumes
 # https://learn.microsoft.com/en-us/azure/virtual-machine-scale-sets/tutorial-use-disks-cli
-
+# Find/create/attach volumes
+
+instanceHostname=\'$(hostname)\'
```

Comment:
I think that the most widely spread "convention" for variable naming is: lower snake case, except for env vars, which should be upper case.